common/djangoapps/student/views.py

```diff
@@ -1290,8 +1290,8 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
                 "success": False,
                 "value": _(
                     "This account has been temporarily locked due to excessive login failures. "
-                    "Try again in {minutes} minute(s).  For security reasons, "
-                    "reseting the password will NOT lift the lockout. Please wait for {minutes} minute(s).").format(
+                    "Try again in {minutes} minutes.  For security reasons, "
+                    "resetting the password will NOT lift the lockout. Please wait for {minutes} minutes.").format(
```

Comment:
What if the lockout was just one minute?
What if the site is translated into a language that handles pluralization differently?

Luckily, `ngettext` (as opposed to `gettext`) has got you covered [1].

[1] https://docs.python.org/2/library/gettext.html#gettext.ngettext